pkgs/development/haskell-modules/make-package-set.nix

```diff
@@ -129,12 +129,9 @@ let
     in buildPackages.stdenv.mkDerivation {
       name = "cabal2nix-${name}";
       nativeBuildInputs = [ buildPackages.cabal2nix-unwrapped ];
-      preferLocalBuild = true;
-      allowSubstitutes = false;
-      phases = ["installPhase"];
       LANG = "en_US.UTF-8";
       LOCALE_ARCHIVE = pkgs.lib.optionalString (buildPlatform.libc == "glibc") "${buildPackages.glibcLocales}/lib/locale/locale-archive";
-      installPhase = ''
+      runCommandLocal = ''
```

Comment:
`runCommandLocal` doesn't work like this, it's a standalone function [defined here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/trivial-builders.nix#L5) and [documented here](https://nixos.org/manual/nixpkgs/stable/#trivial-builder-runCommand).

The way you changed it, `haskellSrc2nix` will cease working, so I'm wondering whether you have actually tested that change.

Reply:
i only tested for compilation, not for execution. maybe we should address the change of `haskellSrc2nix` in a different PR.